### DISCUSSION TITLE
AV Attendance Boost.

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#6855] [Suggestions]
The issue is that not many players seem keen on queuing up for AV. Week after week, it barely pops up 2-3 times during the whole weekend. It's clear we need a change. Here are a couple of ideas:

**1. Sweeten the Deal for AV Queues:**
Let's make joining AV more enticing. How about doubling the Arena Points (from daily win) and Honor rewards(from first win) during AV weekends? That could really get more players excited to join in.

**2. Replace AV as filler for IOC**
AV hardly ever pops up, so why keep it in the rotation twice? Once for AV Call to Arms and then again as a filler for IOC just doesn't make sense. Let's consider swapping it out with something else when it's not popping enough. This way, we keep things fresh and players stay engaged without forcing a battleground that's losing interest. Let AV CTA be AV CTA, and IoC, switch it for something else, maybe we can have a vote about it.

**Comment by balleny** [comment]
or:
disable CTA and have proper random bg queue instead rotating randomly through the list

**Comment by Wayran-is-typing** [comment]
I think x3 or x4 exp bonus (doesn't matter if you lose or win the bg) for low level players/alts could make the pops more often on AV CtA

**Comment by rakrak1775** [comment]
bump

**Comment by TotallyNotMyAccount** [comment]
Remove this BG from the CTA rotation until the server has the population to actually fill it. Being in a raid while the only AV of the day is happening is a miserable feeling, considering you are losing out on CTA arena points.

**Reply by Alebjorn** [reply]
That's completely unfair to the players that like to play AV.  A reduced player requirement could be implemented.
